src/cwe_checker_lib/src/intermediate_representation/def.rs

```diff
@@ -86,6 +88,16 @@ impl Term<Def> {
     }
 }
 
+impl fmt::Display for Def {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Def::Load { var, address } => write!(f, "{} = @{}", var, address),
+            Def::Store { address, value } => write!(f, "@{} = {}", address, value),
```

Comment:
I prefer C-like syntax, i.e. `*{}` instead of `@{}` in Load and Store instructions. Maybe we should also use `:=` instead of `=` to make it explicitly clear that the right side is assigned to the left side. Although this would not be C-like syntax...

Reply:
Another possibility would be `... := Load from ...` for loads and `Store at ... := ...` for stores. This variant would be easier to spot when scrolling over larger amounts of code.